frontend/src/app/testResults/TestResultPrintModal.tsx

```diff
@@ -84,7 +92,13 @@ export const DetachedTestResultPrintModal = ({
       overlayClassName="sr-test-results-modal-overlay"
       contentLabel="Printable test result"
     >
-      {buttonGroup}
+      <div
+        className="display-flex flex-align-center maxw-tablet grid-container patient-header"
+        style={{ paddingLeft: "0rem", paddingRight: "0rem" }}
```

Comment:
USWDS has some [utility classes](https://designsystem.digital.gov/utilities/margin-and-padding/) for this. `padding-x-0` should work!

Reply:
I knew that couldn't be the best way of doing that 😁 Looks great (528df12)